testTypeSimple.py

```diff
@@ -48,6 +48,9 @@ def cmd_line_args(self, *, for_template: bool = False) -> list[str]:
     def _create_setup_operation_get_cancel_action_cmd(self) -> str:
         return "killall python3"
 
+    def _get_server_listen_protocol(self) -> str:
+        return "tcp"
```

Comment:
I was wondering, whether the parent implementation should default to `return "tcp"`. Then most subclasses wouldn't need to override the function, it also seems a good default for a "Server"Task.

If we ever add a future `ServerTask` that does not have an actual port, then that one can override the function to `return None`.